cypress/support/fragments/requests/edit-request.js

```diff
@@ -0,0 +1,152 @@
+import { including } from '@interactors/html';
+import { Button, KeyValue, Section, Select, TextField } from '../../../../interactors';
+import Requests from './requests';
+
+const paneResultsSection = Section({ id: 'pane-results' });
+const requestPreviewSection = Section({ id: 'instance-details' });
+const actionsButton = requestPreviewSection.find(Button('Actions'));
+const editRequestButton = Button('Edit');
+const saveAndCloseButton = Button('Save & close');
+const closeButton = Button({ icon: 'times' });
+const requestExpirationDateInput = TextField({ id: 'requestExpirationDate' });
+const fulfillmentPreferenceSelect = Select({ name: 'fulfilmentPreference' });
+const pickupServicePointSelect = Select({ name: 'pickupServicePointId' });
+const holdShelfExpirationDateInput = TextField({ name: 'holdShelfExpirationDate' });
+const deliveryTypeAddressTypeId = Select({ name: 'deliveryAddressTypeId' });
+const requestExpirationDateKeyValue = KeyValue('Request expiration date');
+const holdShelfExpirationDateKeyValue = KeyValue('Hold shelf expiration date');
+const pickupServicePointKeyValue = KeyValue('Pickup service point');
+
+
+export default {
+  servicePoint: 'Circ Desk 1',
+
+  fulfillmentPreference: {
+    HOLD_SHELF: 'Hold Shelf',
+    DELIVERY: 'Delivery'
+  },
+
+  requestStatuses: {
+    NOT_YET_FILLED: 'Open - Not yet filled',
+    IN_TRANSIT: 'Open - In transit',
+    AWAITING_PICKUP: 'Open - Awaiting pickup',
+    AWAITING_DELIVERY: 'Open - Awaiting delivery',
+  },
+
+  expirationDates: [
+    { formValue: '2050-01-01', uiValue: '1/1/2050' },
+    { formValue: '2050-01-02', uiValue: '1/2/2050' },
+    { formValue: '2050-01-03', uiValue: '1/3/2050' },
+    { formValue: '2050-01-04', uiValue: '1/4/2050' },
+    { formValue: '2050-01-05', uiValue: '1/5/2050' },
+  ],
+
+  updateRequestApi(requestData, status) {
+    return cy.changeItemRequestApi({ ...requestData, status });
```

Comment:
let's move changeItemRequestApi from cypress\support\api\requests.js to cypress\support\fragments\requests\edit-request.js and rename it to updateRequestApi